cpp/velox/shuffle/VeloxShuffleReader.cc

```diff
@@ -450,7 +451,7 @@ std::shared_ptr<ColumnarBatch> VeloxSortShuffleReaderDeserializer::deserializeTo
   return std::make_shared<VeloxColumnarBatch>(std::move(rowVector));
 }
 
-class VeloxRssSortShuffleReaderDeserializer::VeloxInputStream : public facebook::velox::ByteInputStream {
+class VeloxRssSortShuffleReaderDeserializer::VeloxInputStream : public facebook::velox::GlutenByteInputStream {
```

Comment:
I make a workaround here because some API is removed by https://github.com/facebookincubator/velox/pull/10717 and https://github.com/facebookincubator/velox/pull/10761/files

Can you help to refactor it when you are free? @marin-ma Thanks!

Reply:
cc: @zhli1142015